medusa-config.js

```diff
@@ -1,133 +1,14 @@
-const dotenv = require("dotenv");
-const { Modules } = require("@medusajs/modules-sdk");
+import { loadEnv, defineConfig } from '@medusajs/utils'
 
-let ENV_FILE_NAME = "";
-switch (process.env.NODE_ENV) {
-  case "production":
-    ENV_FILE_NAME = ".env.production";
-    break;
-  case "staging":
-    ENV_FILE_NAME = ".env.staging";
-    break;
-  case "test":
-    ENV_FILE_NAME = ".env.test";
-    break;
-  case "development":
-  default:
-    ENV_FILE_NAME = ".env";
-    break;
-}
+loadEnv(process.env.NODE_ENV, process.cwd())
```

Comment:
Did we agree on making these optional? Or would this be a change made later?

Reply:
Yeah, going to create a PR for it separately.